projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -104,6 +112,9 @@ export class MultiSelectComponent<V> implements AfterContentInit, OnChanges {
   @Input()
   public showBorder: boolean = false;
 
+  @Input()
+  public enableSearch: boolean = false;
```

Comment:
I'm really questioning the need here - it seems to me that if we've got so many items in a multi-select that we need to search, we're probably doing something wrong.

Reply:
I think its a natural progression for a component like this. In cases where we might have a long list, we want to support search.

Reply:
Discussed offline, yes there is